packages/manager/apps/telecom/src/app/telecom/pack/slots/domain/index.js

```diff
@@ -0,0 +1,29 @@
+import angular from 'angular';
+import ngOvhTelecomUniverseComponents from '@ovh-ux/ng-ovh-telecom-universe-components';
+import '@uirouter/angularjs';
+import '@ovh-ux/ng-translate-async-loader';
+import 'angular-translate';
+import 'ovh-api-services';
+
+import controller from './domain.controller';
+import template from './domain.html';
+
+const moduleName = 'PackDomain';
```

Comment:
```suggestion
const moduleName = 'ovhManagerTelecomPackDomain';
```

And for all `moduleName`.